Makefile

```
@@ -0,0 +1,36 @@
+
+clean-all: clean clean-playgrounds
+
+clean-playgrounds: clean-petclinic-maven clean-petclinic-gradle clean-notification-gradle clean-feature-maven
+
+spotless:
+	./gradlew spotlessApply
+
+publish-local:
+	./gradlew publishNexusPublicationMavenLocal
+
+all: build-starter build-playground publish-local spotless
```

checkmake (0.3.2)

[warning] Target "all" should be declared PHONY.

(phonydeclared)